### PR TITLE
ROD-45 What Is Application For page

### PR DESCRIPTION
**Pull request by vivekkumar-ho**
## What? 
[ROD-45](https://collaboration.homeoffice.gov.uk/jira/browse/ROD-45) - What is the application for page
## Testing?
Local and branch testing
## Check list

- [x] I have reviewed my own pull request for linting issues (e.g. adding new lines)
- [ ] I have written tests (if relevant)
- [x] I have created a JIRA number for my branch
- [x] I have created a JIRA number for my commit
- [x] I have followed the chris beams method for my commit https://cbea.ms/git-commit/
here is an [example commit](https://github.com/UKHomeOfficeForms/hof/commit/810959f391187c7c4af6db262bcd143b50093a6e)
- [x] Ensure drone builds are green especially tests
- [x] I will squash the commits before merging

### CODE REVIEW

File: apps/cancel-request/sections/summary-data-sections.js [hunk 1]
```diff
@@ -26,6 +26,10 @@ module.exports = {
       step: '/cancel-request-who-representing',
       field: 'cnc-who-is-representing'
     },
+    {
+      step: '/cancel-request-application',
+      field: 'cnc-reason-for-application'
```
Comment (vinodhasamiyappanHO) [inline apps/cancel-request/sections/summary-data-sections.js:31]: summary page label for this field is different. please add a entry in pages.json 
```
 "confirm": {
    "fields": {
```

File: apps/cancel-request/index.js [hunk 2]
```diff
@@ -60,20 +60,22 @@ module.exports = {
       next: '/cancel-request-application'
     },
     '/cancel-request-application': {
+      fields: ['cnc-reason-for-application'],
       forks: [
         {
           target: '/cancel-request-visa-type',
+          continueOnEdit: true,
```
[the remark this answers is not in the document]
Comment (vivekkumar-ho) [reply apps/cancel-request/index.js:67]: Sure. Probably I wasn't aware we can put this globally on all forks. Will update this. Just to ensure I get this right, something like below

    '/cancel-request-application': {
      fields: ['cnc-reason-for-application'],
      continueOnEdit: true,
      forks: [
        {
          target: '/cancel-request-visa-type',
          condition: {
            field: 'cnc-reason-for-application',
            value: 'visa'
          }
        },
        {
          target: '/cancel-request-further-leave',
          condition: {
            field: 'cnc-reason-for-application',
            value: 'leave-to-remain'
          }
        }
      ],
      next: '/cancel-request-reference-number'
    },

File: apps/cancel-request/sections/summary-data-sections.js (hunk 1 above)
Comment (vivekkumar-ho) [reply apps/cancel-request/sections/summary-data-sections.js:34]: Right, thanks for highlighting this. I was aware that the field name is different and I thought we will have to handle this in `parse` method :D . Will update this in the translation

Comment (vivekkumar-ho) [reply apps/cancel-request/sections/summary-data-sections.js:34]: Updated.

File: apps/cancel-request/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (vivekkumar-ho) [reply apps/cancel-request/index.js]: Updated.

**Review by vinodhasamiyappanHO — APPROVED** [review]
Looks good to me

**Pull request merged**